### ISSUE TITLE
Change design of i/o ports

### ISSUE THREAD

**Issue by AlexanderKnueppel** [AlexanderKnueppel/ContractIDE#2]
![image](https://user-images.githubusercontent.com/263321/45475834-c1604500-b73d-11e8-8c83-03e5f2c5f1f6.png)

Maybe it would be a good idea to somehow visualize whether a port is an input-port OR an output-port... any ideas?

**Comment by hfenske33** [comment]
Maybe with symbols inside the rectangle
![45475834-c1604500-b73d-11e8-8c83-03e5f2c5f1f6](https://user-images.githubusercontent.com/32099069/45741201-4e504600-bbf7-11e8-9098-0b6a2ba12276.png)
...

**Comment by AlexanderKnueppel** [comment]
Yeah that's a good and simple idea for a first version. We should go for that!

**Comment by AlexanderKnueppel** [comment]
We should distinguish between input- and output-ports + it is not intuitive to freely move ports around. Maybe attach ports to left or right of the component depending on their type. We could implement mechanisms for a set of layouting algorithms.

**Comment by AlexanderKnueppel** [comment]
The current version looks like this:

![image](https://user-images.githubusercontent.com/263321/48932498-9132e080-eefc-11e8-9d1c-9008bc3669d0.png)

There are a couple of issues.
* Text and symbols may overlap (there should be some space between them)
* e (formerly for external) should be o (for output)
* I find the picture above more appealing (i.e., bigger and with more emphasis)
* Maybe input should remain on the left side and output could indeed be positioned on the right side

The corresponding code is in de.tu-bs.ccc.contracting.core.features.AddPortFeature.java
